### ISSUE TITLE
Tests fail with `Invalid formatter u` on OCaml 5

### ISSUE THREAD

**Issue by Leonidas-from-XIV** [mirage/conan#20]
While building the revdeps for Mirage, [Conan fails on OCaml 5.0](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/47c6cb05a1b1f4bd14348f0cc8477ba9ee2a879c):

```
lint (assert false / exception): FAIL

When given the input:

    "CI\175\226j\154\237\132&\140\1974\004O\018\227\209\237ZGm\202\202D\127\020\180;\019\173Z\187\151%\149\223+\244n\156c\027\148"
    
the test threw an exception:

    Invalid_argument("Invalid formatter u")
    Raised at Stdlib.invalid_arg in file "stdlib.ml", line 30, characters 20-45
    Called from Conan__Fmt.go in file "src/fmt.ml", line 618, characters 24-44
    Called from Conan__Fmt.ty_of_string in file "src/fmt.ml", line 638, characters 18-30
    Called from Conan__Tree.key_of_ty in file "src/tree.ml", line 195, characters 21-50
    Called from Conan__Tree.format_of_ty in file "src/tree.ml", line 242, characters 18-38
    Called from Conan__Process.process_fmt in file "src/process.ml", line 16, characters 31-51
    Called from Conan__Process.process.(fun) in file "src/process.ml", line 53, characters 25-54
    Called from Conan__Process.descending_walk.iter in file "src/process.ml", line 131, characters 8-68
    Called from Conan__Process.descending_walk.iter.(fun) in file "src/process.ml", line 134, characters 12-69
    Called from Conan__Process.descending_walk.iter.(fun) in file "src/process.ml", line 134, characters 12-69
    Called from Conan_string.run in file "string/conan_string.ml", line 148, characters 25-79
    Called from Dune__exe__Fuzz.(fun) in file "fuzz/fuzz.ml", line 33, characters 8-38
    Called from Crowbar.gen_apply.go.(fun) in file "src/crowbar.ml", line 325, characters 16-19
```

I tried submitting a PR but I don't quite understand where the `u` is coming from.

**Comment by dinosaure** [comment]
Fixed by #21

**Comment by Leonidas-from-XIV** [comment]
Excellent, thanks for the super fast fix!
